app/controllers/application_controller.rb

```diff
@@ -1,4 +1,17 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  include Pundit::Authorization
+
+  rescue_from Pundit::NotAuthorizedError, with: :employee_not_authorized
+
+  def pundit_user
+    current_employee
+  end
+
+  private
+
+  def employee_not_authorized
+    redirect_to root_path, notice: "You don't have access to this kudo"
```

Comment:
I woudl suggest some more generic message, since those policies might not be used to kudos only :)

```suggestion
    redirect_to root_path, notice: "You are not authorized to perform this action."
```

Reply:
changed 🙂